### PR TITLE
feature-44-resume-detail

### PR DESCRIPTION
**Pull request by niver-copa-jalafund**
Hi guys, I was send you my progress with the US-44, I hope, you let me know, if you find any troubles.

### CODE REVIEW

File: src/api/Jalasoft.TeamUp.Resumes.API/Controllers/GetResumes.cs [hunk 1]
```diff
@@ -23,10 +24,11 @@ public GetResumes(IResumesService resumesService)
...
         [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Resume), Description = "Successful response")]
         public IActionResult Run(
-            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req)
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "v1/resumes/{id}")] HttpRequest req)
         {
-            var resumes = this.resumesService.GetResumes();
-            return new OkObjectResult(resumes);
+            Guid id = Guid.Parse(req.Query["id"]);
+            var result = this.resumesService.GetResume(id);
+            return new OkObjectResult(result);
```
Comment (rodrigo-baldiviezo-jalafund) [inline src/api/Jalasoft.TeamUp.Resumes.API/Controllers/GetResumes.cs:35]: I think you need to create a new file "GetResume", and put your code there

File: src/core/Jalasoft.TeamUp.Resumes.Core.Interfaces/IResumesService.cs [hunk 2]
```diff
@@ -1,9 +1,10 @@
 ﻿namespace Jalasoft.TeamUp.Resumes.Core.Interfaces
 {
+    using System;
     using Jalasoft.TeamUp.Resumes.Models;
 
     public interface IResumesService
     {
-        Resume[] GetResumes();
+        public Resume GetResume(Guid id);
```
Comment (rodrigo-baldiviezo-jalafund) [inline src/core/Jalasoft.TeamUp.Resumes.Core.Interfaces/IResumesService.cs:8]: Add your functionality, but don't delete other

File: src/core/Jalasoft.TeamUp.Resumes.Core/ResumesService.cs [hunk 3]
```diff
@@ -14,9 +14,9 @@ public ResumesService(IResumesRepository resumesRepository)
             this.resumesRepository = resumesRepository;
         }
 
-        public Resume[] GetResumes()
+        public Resume GetResume(Guid id)
```
Comment (rodrigo-baldiviezo-jalafund) [inline src/core/Jalasoft.TeamUp.Resumes.Core/ResumesService.cs:17]: same here, add your functionallity, don't delete

**Review by rodrigo-baldiviezo-jalafund — COMMENTED** [review]
just create a new file in Controllers, in other layers just add your code, don't delete other code :-)

File: src/core/Jalasoft.TeamUp.Resumes.Core/ResumesService.cs (hunk 3 above)
Comment (niver-copa-jalafund) [reply src/core/Jalasoft.TeamUp.Resumes.Core/ResumesService.cs:17]: solved

File: src/core/Jalasoft.TeamUp.Resumes.Core.Interfaces/IResumesService.cs (hunk 2 above)
Comment (niver-copa-jalafund) [reply src/core/Jalasoft.TeamUp.Resumes.Core.Interfaces/IResumesService.cs:8]: solved

**Comment by martin-callejas-jalafund** [comment]
please close the PR and rename your local branch to follow conventions (see wiki)

**Pull request closed without merging**